backend/lib/edgehog/containers/deployment_ready_action.ex

```diff
@@ -0,0 +1,67 @@
+#
+# This file is part of Edgehog.
+#
+# Copyright 2024 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+defmodule Edgehog.Containers.DeploymentReadyAction do
+  @moduledoc false
+  use Edgehog.MultitenantResource,
+    domain: Edgehog.Containers
+
+  alias Edgehog.Containers.ManualActions
+
+  actions do
+    defaults [:read, :destroy]
+
+    create :create_deployment do
+      accept [:action_type]
+
+      argument :action_arguments, :map
+
+      argument :deployment, :map
+
+      change manage_relationship(:deployment, type: :create)
```

Comment:
this will call the primary `:create` action on the deployment, i.e., it will not automatically deploy the release on the device, to do so you have to manually call the deploy create action

```suggestion
      change manage_relationship(:deployment, on_no_match: {:create, :deploy}, on_match: :ignore)
```

Reply:
I was doing that manually [here](https://github.com/edgehog-device-manager/edgehog/pull/728/files#diff-040dec3573e5fdd58268cc6a8a20f9c53d3a78f00e238e72edbd7fa1d2e765efR48), but it may be a better idea to call the correct action on the deployment, right